docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -83,4 +83,18 @@ corresponding CDAP types.
 | int16/int34/int64/long | Long      |
 | String                 | String    |
 | Array                  | Array     |
-| Record                 | Record    |
\ No newline at end of file
+| Record                 | Record    |
+
+Data Duplication issues with Zendesk APIs
+----------
+This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
```

Comment:
Is this also applicable to the source?

Reply:
yes, it is both for Source and multisource